rundeckapp/src/main/groovy/org/rundeck/app/data/providers/GormUserDataProvider.groovy

```diff
@@ -309,4 +312,41 @@ class GormUserDataProvider implements UserDataProvider {
     def getSessionIdRegisterMethod() {
         configurationService.getString(SESSION_ID_METHOD, 'hash')
     }
+
+    /**
+     Checks if login name case sensitivity is enabled.
+     @return {@code true} if login name case sensitivity is enabled, {@code false} otherwise.
+     */
+    def isLoginNameCaseSensitiveEnabled(){
+        return configurationService?.getBoolean(NAME_CASE_INSENSITIVE_ENABLED,false)
```

Comment:
the feature check should use the `featureService.featurePresent` instead of direct check with the configurationService